modules/ROOT/pages/_partials/block-caveats.adoc

```diff
@@ -37,7 +37,7 @@ For documentation on later Inter-Sync Gateway Replication features see {xref-sgw
 .Context Clarification
 [CAUTION]
 --
-This content relates only to replications run using the latest version of Inter-Sync Gateway Replication (release 2.8+).
+This content relates only to Inter-Sync Gateway replication.
```

Comment:
```suggestion
This content relates only to Inter-Sync Gateway replications using the version of Sync Gateway 2.8+.
```
GENERAL : Don't treat "Inter- Sync Gateway replication" to be synonymous to what's introduced in 2.8. Treat it as a capability

Reply:
GENERAL: Unless in headings, please do not capitalize "I" or "R" in "Inter-Sync Gateway Replication"...others seems like a title/name..so if its used in middle, you should just say "inter-Sync Gateway replication" (only the proper noun is capitalized...this is what we are doing on our couchbase web pages)

Reply:
GENERAL: For consideration: In fact I am wondering if we need to keep calling out "Inter-SGW replications" every where in a page which is only  about that capability , so we should just use "replications" where it makes sense. Otherwise, it seems like we are "naming" this thing..which we are not

 Its similar to "peer-to-peer" replications - we don't keep stating p2p everywhere in that page. We just refer to that as "replication" and in context is the page, it makes sense that its "peer-to-peer"

Reply:
Ok, that makes sense